thirdeye/thirdeye-spi/src/main/java/org/apache/pinot/thirdeye/detection/performance/PerformanceMetrics.java

```diff
@@ -0,0 +1,136 @@
+package org.apache.pinot.thirdeye.detection.performance;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+
+/**
+ * The performance metrics of a detection based on the given anomalies
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PerformanceMetrics {
+
+  @JsonProperty
+  private PerformanceMetric totalAnomalies;
+
+  @JsonProperty
+  private PerformanceMetric responseRate;
+
+  @JsonProperty
+  private PerformanceMetric precision;
+
+  @JsonProperty
+  private PerformanceMetric recall;
+
+  public PerformanceMetric getTotalAnomalies() { return totalAnomalies; }
+
+  public PerformanceMetric getResponseRate() { return responseRate; }
+
+  public PerformanceMetric getPrecision() { return precision; }
+
+  public PerformanceMetric getRecall() { return recall; }
+
+  /**
+   * Builder for performance metrics
+   */
+  public static class Builder {
+    private long respondedAnomalies, truePos, falsePos, falseNeg, numAnomalies = 0L;
+    private boolean includeTotalAnomalies, includeResponseRate, includePrecision, includeRecall = false;
+    private final String NOT_CLASSIFIED = "NONE";
+    private final String TRUE_POSITIVE = "TRUE_POSITIVE";
+    private final String FALSE_POSITIVE = "FALSE_POSITIVE";
+    private final String FALSE_NEGATIVE = "FALSE_NEGATIVE";
+
+    public Builder (List<String> statusClassifications) {
+      statusClassifications.stream()
+          .forEach(classification -> {
+            this.numAnomalies++;
+            if (!classification.equals(NOT_CLASSIFIED)) {
```

Comment:
consider (!NOT_CLASSIFIED.equals(classification)) to avoid possible NPE

Reply:
great catch, I've updated